core/domain/suggestion_registry.py

```diff
@@ -156,11 +157,27 @@ def validate(self):
                 'Expected author_id to be a string, received %s' % type(
                     self.author_id))
 
-        if not isinstance(self.final_reviewer_id, python_utils.BASESTRING):
-            if self.final_reviewer_id:
+        if (
+                self.author_id is not None and
+                not user_id_migration.verify_user_id_correct(self.author_id)
```

Comment:
@vojtechjelinek can you confirm that this is OK please? Will user_id_migration still be around after the migration / should we move this validation function to a more permanent place?

(I think you missed my comment [here](https://github.com/oppia/oppia/pull/9511#discussion_r439705012).)

Reply:
(Ditto below)

Reply:
It is okay since this is going to the release and I don't want to do any modifications to the migration code. We can move it away later. The migration will stay on the prod at least for month and when I will be removing the code I will move this verification elsewhere.

Reply:
SGTM. Thanks!